README.md

```diff
@@ -146,3 +146,9 @@ LaunchDarkly uses the [SLSA framework](https://slsa.dev/spec/v1.0/about) (Supply
 
 [ghrelease-badge]: https://img.shields.io/github/release/launchdarkly/ldcli.svg
 [ghrelease-link]: https://github.com/launchdarkly/ldcli/releases/latest
+
+## Dev Server
+
+LaunchDarkly provides a command line interface (CLI), which includes a `dev-server` command that you can use to start a local server and retrieve flag values from a LaunchDarkly source environment. This development server supports a single variation value for each flag, which you can override as needed. This means you can test your code locally, and you do not need to coordinate with other developers in your organization who are using the same LaunchDarkly source environment.
```

Comment:
I think we want this at line 98: we're just documenting the `dev-server` command and linking out to additional documentation about it. We also can remove the "LaunchDarkly provides a command line interface (CLI), which includes a" bit. If someone is reading this readme, they already know that.